### PR TITLE
Feature/sidebar

### PR DESCRIPTION
**Pull request by mdroidian**
srsly best

🙀🙀🙀🙀🙀

@navpreetvirk

### CODE REVIEW

File: file.file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is file
```
Comment (mdroidian) [inline file.file.txt:1]: is it tho?

Comment (navpreetvirk) [reply file.file.txt:1]: this is good

Comment (navpreetvirk) [reply file.file.txt:1]: purrr fect

**Review by navpreetvirk — COMMENTED** [review]
looks good

**Pull request closed without merging**